meta/parse.pl

```diff
@@ -2598,6 +2598,9 @@ sub ProcessStructValueType
     return "SAI_ATTR_VALUE_TYPE_BOOL"           if $type eq "bool";
     return "SAI_ATTR_VALUE_TYPE_IPV6"           if $type eq "sai_ip6_t";
     return "SAI_ATTR_VALUE_TYPE_UINT8"          if $type eq "sai_uint8_t";
+    return "SAI_ATTR_VALUE_TYPE_UINT32"         if $type eq "sai_stat_id_t";
+    return "SAI_ATTR_VALUE_TYPE_UINT64"         if $type eq "uint64_t";
+    return "SAI_ATTR_VALUE_TYPE_TWAMP_STAT_DATA" if $type eq "sai_twamp_session_stat_data_t";
```

Comment:
please align all if's in this functions so they start at the same column

Reply:
Thanks for review, I have aligned all to make them to start at the same column.